diff --git a/build/src/frida.js b/build/src/frida.js
index 0b047472199e9151691c52f03ffd36a85952234c..ee9395ae0a8307d8e5a9a31f8663d5d612683b0e 100644
--- a/build/src/frida.js
+++ b/build/src/frida.js
@@ -1,15 +1,8 @@
-import bindings from "bindings";
 import util from "util";
 import { Minimatch } from "minimatch";
 import { Duplex } from "stream";
 const { inspect } = util;
-const binding = bindings({
-    bindings: "frida_binding",
-    try: [
-        ["module_root", "build", "bindings"],
-        [process.cwd(), "bindings"],
-    ]
-});
+const binding = require('../../build/frida_binding.node');
 var MessageTypeImpl;
 (function (MessageTypeImpl) {
     MessageTypeImpl["Send"] = "send";
