diff --git a/node_modules/frida-compile/dist/cli.js b/node_modules/frida-compile/dist/cli.js
index 9243bf6..a9568d7 100644
--- a/node_modules/frida-compile/dist/cli.js
+++ b/node_modules/frida-compile/dist/cli.js
@@ -46,9 +46,9 @@ async function main() {
         outputFormat: opts.outputFormat,
         bundleFormat: opts.bundleFormat,
         typeCheck: opts.typeCheck,
-        sourceMaps: opts.sourceMaps
-            ? frida.SourceMaps.Included
-            : frida.SourceMaps.Omitted,
+        sourceMaps: opts.noSourceMaps
+            ? frida.SourceMaps.Omitted
+            : frida.SourceMaps.Included,
         compression: opts.compress
             ? frida.JsCompression.Terser
             : frida.JsCompression.None,
